frontend/packages/metal3-plugin/src/components/host.tsx

```diff
@@ -1,57 +1,73 @@
 import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as classNames from 'classnames';
+import { sortable } from '@patternfly/react-table';
 
-import { getName, getNamespace } from '@console/shared';
-// TODO(jtomasek): update import once models are moved to console-shared package
-// import { MachineModel, NodeModel } from '@console/internal/models';
-import { referenceForModel } from '@console/internal/module/k8s';
+import { getName, getNamespace, getMachineNode } from '@console/shared';
+import { MachineModel, NodeModel } from '@console/internal/models';
 
-import {
-  ListHeader,
-  ColHead,
-  List,
-  ListPage,
-  ResourceRow,
-} from '@console/internal/components/factory';
-
-import { ResourceLink } from '@console/internal/components/utils';
-// import { WithResources } from '@console/shared';
+import { MultiListPage, Table, TableRow, TableData } from '@console/internal/components/factory';
+import { ResourceLink, Kebab } from '@console/internal/components/utils';
+import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
 
 import { BaremetalHostModel } from '../models';
-import { getHostBMCAddress } from '../selectors';
+import { getHostBMCAddress, getHostMachine } from '../selectors';
+import { BaremetalHostRole } from './host-role';
 import MachineCell from './machine-cell';
 
-// const nameColumnClasses = 'col-lg-2 col-md-4 col-sm-6 col-xs-6';
-// const statusColumnClasses = 'col-lg-2 col-md-4 hidden-sm hidden-xs';
-// const machineColumnClasses = 'col-lg-3 visible-lg';
-// const roleColumnClasses = 'col-lg-2 visible-lg';
-// const addressColumnClasses = 'col-lg-2 visible-lg';
-const columnClasses = 'col-sm-4';
-
-const HostHeader = (props: React.ComponentProps<typeof ColHead>) => (
-  <ListHeader>
-    <ColHead {...props} className={columnClasses} sortField="metadata.name">
-      Name
-    </ColHead>
-    {/* <ColHead {...props} className={statusColumnClasses}>
-      Status
-    </ColHead> */}
-    <ColHead {...props} className={columnClasses}>
-      Machine
-    </ColHead>
-    {/* <ColHead {...props} className={roleColumnClasses}>
-      Role
-    </ColHead> */}
-    <ColHead {...props} className={columnClasses} sortField="spec.bmc.address">
-      Management Address
-    </ColHead>
-  </ListHeader>
-);
+const tableColumnClasses = [
+  classNames('pf-m-5-col-on-lg', 'pf-m-8-col-on-md', 'pf-m-12-col-on-sm'),
```

Comment:
Sorry if we're asking you to code to a moving target, but we removed these classes. They aren't standard Patternfly classes. See #1707

Reply:
Done